zenoh/src/prelude.rs

```diff
@@ -36,7 +36,7 @@
 //! ```
 
 // Reexport API in flat namespace
-pub(crate) mod flat {
+pub mod flat {
```

Comment:
here `pub(crate)` (or better `pub(self)`?) should be restored: these namespaces are technical, they are not part of public API